script/lib/native_tests.py

```diff
@@ -0,0 +1,274 @@
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+
+SOURCE_ROOT = os.path.abspath(
+    os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+VENDOR_DIR = os.path.join(SOURCE_ROOT, 'vendor')
+PYYAML_LIB_DIR = os.path.join(VENDOR_DIR, 'pyyaml', 'lib')
+sys.path.append(PYYAML_LIB_DIR)
+import yaml  #pylint: disable=wrong-import-position,wrong-import-order
+
+
+class Verbosity:
+  CHATTY = 'chatty'  # stdout and stderr
+  ERRORS = 'errors'  # stderr only
+  SILENT = 'silent'  # no output
+
+  @staticmethod
+  def get_all():
+    return Verbosity.__get_all_in_order()
```

Comment:
Doesn't seem to be used; does this need to exist / need to be public?

Reply:
`.get_all` is used in the `script/native-tests.py` to provide a list of choices for the `--verbosity` arg.

Reply:
I don't want to make `get_all_in_order()` public because it would reveal to much information, and don't want to rename it to simply `get_all` either, because it would hide an important property of a returned data.